pullrequest.rst

```diff
@@ -105,23 +105,23 @@ Here is a quick overview of you can contribute to CPython on GitHub:
 
 1.  If an Issue doesn't exist, `create an Issue`_ that describes your change
 
-2.  Prepare your system by :ref:`setup`
+2.  :ref:`Get started <setup>`_ and set up your system
```

Comment:
Erp! Thank you! I thought I might have messed up some of the references, but got caught up in rST particulars. Thanks -- will check that more-closely next time. For better of worse, Sphinx didn't throw any errors for me locally.